navigator/backend/src/main/scala/com/digitalasset/navigator/model/Model.scala

```diff
@@ -194,11 +195,18 @@ final case class Template(
     id: DamlLfIdentifier,
     choices: List[Choice],
     key: Option[DamlLfType],
+    implementedInterfaces: Set[DamlLfIdentifier] = Set.empty,
 ) extends DamlLfNode {
   def topLevelDecl: String = id.qualifiedName.toString()
   def parameter: DamlLfTypeCon = DamlLfTypeCon(DamlLfTypeConName(id), DamlLfImmArraySeq())
 }
 
+/** Interfaces. */
+final case class AstInterface(
```

Comment:
Since the iface library name has changed you could name this Interface

Reply:
Sure renamed to "interface" in navigator code but not the one in library